lib/types/multipart.js

```diff
@@ -258,7 +265,9 @@ function Multipart(boy, cfg) {
           curField = undefined;
           if (buffer.length)
             buffer = decodeText(buffer, 'binary', charset);
-          boy.emit('field', fieldname, buffer, false, truncated, encoding, contype);
+
+          var hdrs = Object.keys(additionalHeaders).length ? additionalHeaders : null;
```

Comment:
Yeah, an empty object might be nice there, since it would avoid two null checks I suppose.  I think I'll just pass in the object if that's cool with you.